cluster/addons/dashboard/dashboard-controller.yaml

```diff
@@ -31,12 +36,26 @@ spec:
             memory: 100Mi
         ports:
         - containerPort: 9090
```

Comment:
why didn't this port change if everything is shifting to 8443?

Reply:
It should be changed. This option should expose port `9090` of this container, right? Is this overridden by using `expose` option in dockerfile? This container will actually only expose port `8443`.

Reply:
Missed it before. Fixed now.